README.md

```diff
@@ -129,6 +131,18 @@ which will print:
 {"service":{"name":"service name"},"@timestamp":"2020-05-14T10:58:30.780+01:00","log":{"level":"error"},"event":{"action":"HTTP request"},"trace":{"id":"126bb6fa-28a2-470f-b013-eefbf9182b2d"},"message":"Error 500 in /pets/buy","http":{"request":{"method":"post","url.path":"/pet/buy"},"response":{"status_code":500}}}
 ```
 
+### Log formatting
```

Comment:
I think in this section you should state that the formatter can be obtained using logger.formatter/make a new Twiglet formatter from scratch.

And if you're putting in the example below I think it's better to say "here's an example to configure logging for Sidekiq". This is because different gems might have different ways of configuring themselves and the below example is for Sidekiq.

Reply:
> "here's an example to configure logging for Sidekiq"
+1 will fix!

Reply:
On second thought, I think we should ONLY mention the 2 ways to get a Ruby Logger compatible Formatter on here.

And completely remove the docs on how to integrate with third part applications (can still allude to it though but don't need to have anything specific.

And for your card I think worth moving the documentation to this card here which fixes the current issues with Twiglet and Sidekiq: https://trello.com/c/wpp4pUnc/548-make-sidekiq-output-ecs-compliant-logs-using-twiglet

Reply:
> make a new Twiglet formatter from scratch
Not convinced I agree with this - I think we should encourage that the formatter is only accessed via a logger for a couple of reasons

1) the logger is built to be customizable and creating a new formatter means losing that customization
2) the logger's interface is well documented, and I don't want to document the formatter interface 😛 . Plus I think things are just simpler with only one interface to worry about.

Reply:
Yup I see your point!

Even in scenarios where only the formatter is needed, it is better to use the logger and then get the formatter from the logger anyways. So happy with just putting in that option!